soc/nxp/rw/power.c

```diff
@@ -6,6 +6,7 @@
 #include <zephyr/kernel.h>
 #include <zephyr/pm/pm.h>
 #include <zephyr/init.h>
+#include <zephyr/drivers/gpio.h>
```

Comment:
is it really necessary to have a dependency on GPIO to use this feature the way we are enabling it here? I believe all we are needing to do is set one bit to make the pin input? If we do have a dependency on GPIO then we need to capture it in Kconfig

Reply:
I have changed this to use pinctrl directly